modules/organisation/add-organisation.html

```diff
@@ -5,10 +5,10 @@
 	<div class="form-group">
 	<label for="orgName">Navn på organisasjon</label>
 	<input class="form-control" id="orgName" type="text" ng-model="addOrganisationController.orgName" />
-	<label for="orgShortName" type="text">Kortnavn uten mellomrom</label>
-	<input class="form-control" id="orgShortName" type="text" ng-model="addOrganisationController.orgShortName" />
+	<label for="orgShortName" type="text">Kortnavn uten mellomrom og andre spesialteikn</label>
+    <input class="form-control" id="orgShortName" type="text" pattern="[a-z\x00e6\x00f8\x00e5A-Z\x00c6\x00d8\x00c50-9]{3,}" ng-model="addOrganisationController.orgShortName" />
 	<button type="submit">Legg til</button>
```

Comment:
Hei Torstein!
Fett at du vil bidra. Jeg merger denne innen et par døgn, den er veldig liten og det er fint med sånne kjappe forbedringer.
Hvis jeg har noen motforestillinger er det dette:
Jeg er i grunn ikke fan av å legge validering av slik mer "sikkerhetsmessig" karakter i frontend når det ikke finnes i backend, det kan skape en falsk trygghet. Såvidt jeg kan se betyr dette at man må ha det norske alfabetet(?) og minst tre bokstaver. Det er gode argumenter for å ha en viss validering både backend og frontend på navn som tross alt brukes i url-er, men kan vi få til dette på node-server og?


har du forresten fått testa endringen?

Torkil